code/game/objects/items/weapons/dnascrambler.dm

```diff
@@ -22,6 +22,10 @@
 		if(src.used)
 			return
 
+		if(M.get_species() == "Machine")
+			to_chat(user, "\red You failed to inject [M.name], as they have no DNA to scramble, nor flesh to inject.")
```

Comment:
don't use color macros reeeeeee
or M.name
<span class='warning'>You failed to inject [M], as they have no DNA to scramble, nor flesh to inject.</span>